app/processing/schedule/get-scheduled-settlements.js

```diff
@@ -21,6 +21,9 @@ const getScheduledSettlements = async (started, transaction) => {
     where: {
       completed: null,
       '$settlements.sourceSystem$': SOURCE_SYSTEM.SFI,
+      '$settlements.received$': {
+        [db.Sequelize.Op.lte]: moment(started).subtract(config.settlementWaitTime).toDate()
+      },
```

Comment:
Only construct settlements that have been in database longer than cool down period.  Set to 10 secs for local and 5 minutes in higher environments